physionet-django/project/authorization/access.py

```diff
@@ -0,0 +1,100 @@
+from datetime import datetime
+
+from django.db.models import Q
+
+from events.models import Event, EventDataset
+from project.models import AccessPolicy, DUASignature, DataAccessRequest, PublishedProject
+from user.models import Training, TrainingType
+
+
+def get_accessible_projects(user):
+    """
+    Returns query filter for published projects accessible by a specified user
+    """
+    query = Q(access_policy=AccessPolicy.OPEN) & Q(deprecated_files=False)
+
+    dua_signatures = DUASignature.objects.filter(user=user)
+
+    if user.is_authenticated:
+        query |= Q(access_policy=AccessPolicy.RESTRICTED) & Q(
```

Comment:
We are not excluding deprecated files here, so does this mean that the filter will return deprecated files (despite them being excluded in the earlier step?). I might be misunderstanding how query construction works, so just checking...